Neos.Flow/Documentation/TheDefinitiveGuide/PartIII/PropertyMapping.rst

```diff
@@ -236,52 +236,92 @@ on large collections::
 			'Y-m-d'
 		);
 
-.. admonition:: Property Mapping Configuration in the MVC stack
-
-	The most common use-case where you will want to adjust the Property Mapping Configuration
-	is inside the MVC stack, where incoming arguments are converted to objects.
-
-	If you use Fluid forms, normally no adjustments are needed. However, when programming
-	a web service or an ajax endpoint, you might need to set the ``PropertyMappingConfiguration``
-	manually. You can access them using the ``\Neos\Flow\Mvc\Controller\Argument``
-	object -- and this configuration takes place inside the corresponding ``initialize*Action``
-	of the controller, as in the following example:
-
-	.. code-block:: php
-
-		protected function initializeUpdateAction() {
-			$commentConfiguration = $this->arguments['comment']->getPropertyMappingConfiguration();
-			$commentConfiguration->allowAllProperties();
-			$commentConfiguration
-				->setTypeConverterOption(
-				\Neos\Flow\Property\TypeConverter\PersistentObjectConverter::class,
-				\Neos\Flow\Property\TypeConverter\PersistentObjectConverter::CONFIGURATION_CREATION_ALLOWED,
-				TRUE
-			);
-		}
+Property Mapping Configuration in the MVC stack
```

Comment:
Why did I remove this "admonition" and make it an own headline? Because the section about "Configuring the property mapping process" is already pretty long, and it was not possible to directly link (with a hash) into the pretty elaborate part that covers mapping inside the MVC layer.

Also, this is anyway more of a deeper explanation within a specific context, rather than an "admonition".